app/src/main/java/com/codesoom/assignment/domain/CatProduct.java

```diff
@@ -0,0 +1,25 @@
+package com.codesoom.assignment.domain;
+
+import java.math.BigDecimal;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class CatProduct {
```

Comment:
이 정도로 충분하겠죠. 강아지 관련 상품도 팔게 된다고 해서 `DogProduct`를 만들 순 없잖아요?
```suggestion
public class Product {
```

Reply:
> 이 정도로 충분하겠죠. 강아지 관련 상품도 팔게 된다고 해서 `DogProduct`를 만들 순 없잖아요?

아 그러네요. 흠.. 확장 가능성 고려한다고 그렇게 했는데, 강아지 관련 상품 추가한다고 해서 대동소이하니까 그렇게 할필요가 없겠네요